<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="24505" started-at="2020-04-02T01:22:29 CST" finished-at="2020-04-02T01:22:53 CST">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunners.ThirdRunner@752325ad]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="24505" started-at="2020-04-02T01:22:29 CST" finished-at="2020-04-02T01:22:53 CST">
      <class name="testRunners.ThirdRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:testRunners.ThirdRunner@752325ad]" name="setUpClass" is-config="true" duration-ms="340" started-at="2020-04-02T01:22:29 CST" finished-at="2020-04-02T01:22:29 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunners.ThirdRunner@752325ad]" name="runScenario" duration-ms="24111" started-at="2020-04-02T01:22:29 CST" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2020-04-02T01:22:53 CST">
          <params>
            <param index="0">
              <value>
                <![CDATA["User will create a new account"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Title of your feature"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My account] but found [CREATE AN ACCOUNT]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My account] but found [CREATE AN ACCOUNT]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at stepDefinitions.ThenTest.user_login(ThenTest.java:44)
at âœ½.User login(file:///C:/Users/test.admin/WorkspaceAutomate/SeleniumCase/src/test/resources/featureFiles/CreateAnAccount.feature:11)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:testRunners.ThirdRunner@752325ad]" name="tearDownClass" is-config="true" duration-ms="7" started-at="2020-04-02T01:22:53 CST" finished-at="2020-04-02T01:22:53 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunners.ThirdRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
